samples/ngIdentity/ngIdentity.Server/Program.cs

```diff
@@ -0,0 +1,92 @@
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication(IdentityConstants.ApplicationScheme)
+    .AddIdentityCookies()
+    .ApplicationCookie!.Configure(opt => opt.Events = new CookieAuthenticationEvents()
+    {
+        OnRedirectToLogin = ctx =>
+        {
+            ctx.Response.StatusCode = 401;
+            return Task.CompletedTask;
+        }
+    }); 
+builder.Services.AddAuthorizationBuilder();
+
+builder.Services.AddDbContext<AppDbContext>(
+    options => options.UseInMemoryDatabase("AppDb"));
+
+builder.Services.AddIdentityCore<MyUser>()
+    .AddEntityFrameworkStores<AppDbContext>()
+    .AddApiEndpoints();
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+app.MapIdentityApi<MyUser>();
+app.UseDefaultFiles();
+app.UseStaticFiles();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseHttpsRedirection();
+
+app.MapPost("/logout", async (
```

Comment:
Doesn't the `MapIdentityApi` call register the `logout` endpoint too?
I see that in @halter73 's PR it was registered by the `MapAdditionalIdentityEndpoints` method. What was the reason for not having it as part of the MapIdentityApi? I'm sure there was a reason, just wanted to learn rather than point out something wrong here.

Reply:
@mkArtakMSFT,  the main reason for not including logout is because it could require different steps based on the configuration. Login is straightforward, but depending on whether you are using cookies, tokens, or both (or perhaps include an OAuth2/OIDC flow as well) you may need to logout of multiple services. Users should also take care not to implement it as a POST with no parameters because that could expose the endpoint to cross-site request forgery attacks. The sample shows how to implement it with an empty (not null) body to avoid this.

Reply:
@Rick-Anderson who should I tag for review from the docs side for SPA?

Reply:
@jiayac FYI

Reply:
> @Rick-Anderson who should I tag for review from the docs side for SPA?

@tdykstra  and I will review. Should we move the Readme.MD to the https://github.com/dotnet/AspNetCore.Docs so it's a proper doc?

Reply:
I'd like to make a proper doc but probably needs to be higher quality, the README was just a brain dump so I could use it for accuracy when the related doc is written.